src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1668,16 +1668,16 @@ private static IEnumerable<CommandLineReference> ParseAssemblyReferences(string
                 alias = null;
             }
 
-            List<string> paths = ParseSeparatedPaths(value).Where((path) => !string.IsNullOrWhiteSpace(path)).ToList();
+            string[] paths = ParseSeparatedPaths(value).Where((path) => !string.IsNullOrWhiteSpace(path)).ToArray();
```

Comment:
I'd say `ToList()` is better than `ToArray()` here, in that it avoids the last allocation caused by trimming the collection to so `Capacity == Length`

Reply:
@davkean can you make this change so we can merge?